### PR TITLE
Hide Night Light on unsupported hardware

### PR DESCRIPTION
**Pull request by meisenzahl**
Closes https://github.com/elementary/switchboard-plug-display/issues/272

Bildschirmfoto von 2020-12-06 19 03 49

### CODE REVIEW

**Comment by cassidyjames** [comment]
At first I thought it would be better to just hide the entire tab here. But we do have search terms and everything, so there would be multiple places that need to be hidden. Thoughts on this approach, @elementary/ux ?

File: src/Utils.vala [hunk 1]
```diff
@@ -80,4 +80,23 @@ namespace Display.Utils {
 
         return min_scale;
     }
+
+    public static bool is_night_light_supported () {
+        try {
+            string standard_output;
+            int exit_status;
+            Process.spawn_command_line_sync ("/usr/bin/uname -p",
```
Comment (donadigo) [inline src/Utils.vala:88]: If the intention is to only check if we're running on 64bit, a much simpler check of the size of a void pointer `if (sizeof (void*) == 8)` will suffice.

**Review by donadigo — CHANGES_REQUESTED** [review]

File: src/Utils.vala (hunk 1 above)
Comment (davidmhewitt) [reply src/Utils.vala:88]: Nope, the intention is to check ARM or x86, they can both be either 64-bit or 32-bit.

Comment (donadigo) [reply src/Utils.vala:88]: Alright, good to know.

**Pull request closed without merging**